zenoss.py

```diff
@@ -45,7 +49,8 @@ def __router_request(self, router, method, data=None):
         log.debug('Making request to router %s with method %s', router, method)
         uri = '%s/zport/dmd/%s_router' % (self.__host, ROUTERS[router])
         headers = {'Content-type': 'application/json; charset=utf-8'}
-        response = self.__session.post(uri, data=req_data, headers=headers)
+	import pdb;pdb.set_trace()
+        response = self.__session.post(uri, data=req_data, headers=headers,verify=self.__verify)
```

Comment:
spacing doesn't conform to PEP8

https://www.python.org/dev/peps/pep-0008/#id19